core/embed/trezorhal/stm32u5/secure_aes.c

```diff
@@ -40,14 +40,14 @@ secbool secure_aes_init(void) {
 }
 
 static void secure_aes_load_bhk(void) {
-  TAMP->BKP7R;
-  TAMP->BKP6R;
-  TAMP->BKP5R;
-  TAMP->BKP4R;
-  TAMP->BKP3R;
-  TAMP->BKP2R;
-  TAMP->BKP1R;
   TAMP->BKP0R;
```

Comment:
How does this work if there is no value assigned/read?

Reply:
Loading BHK into SAES key register is using dedicated bus, so that the values being transferred are not accessible to software. This is the way to trigger the transfer.

Reply:
I guess there are some macros that tell the compiler to not optimize these calls. (maybe just using `volatile` is enough?)

Reply:
yeah `volatile` should be enough